rules/linux/lnx_password_policy_discovery.yml

```diff
@@ -0,0 +1,25 @@
+title: Password Policy Discovery
+id: ca94a6db-8106-4737-9ed2-3e3bb826af0a
+status: stable
+description: Detects password policy discovery commands
+author: Ömer Günal, oscd.community
+date: 2020/10/08
+references:
+    - https://attack.mitre.org/techniques/T1201/
```

Comment:
Hello @omergunal! Good to see your are back (:
Could you please remove the ATT&CK link and add a link to the relevant Atomic Red Team test here as well?